include/parakeet/util.h

```diff
@@ -9,13 +9,10 @@
 
 #include <math.h>
 
-#ifndef __linux__
+#if defined(_WIN32)
     #define M_PI   3.14159265358979323846264338327950288
 #endif
 
```

Comment:
Maybe we should put a check to ensure M_PI is defined and generate #error if not? Not sure if the compile will complain later or not?

Reply:
By that logic, we could simply check if M_PI is already defined, and only override it if not.

Reply:
Agreed that's a better solution.

- [ ] Only #define M_PI if it's not already defined.

Reply:
Done